extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -50,7 +50,7 @@ public function getMenuNodes() {
 		$exploreWikiaLabel = wfMessage( 'global-navigation-explore-wikia-link-label');
 
 		if ( $wgLang->getCode() === self::DEFAULT_LANG ) {
-			$hubsNodes = (new NavigationModel(true /* useSharedMemcKey */))->getTree(
+			$hubsNodes = (new NavigationModel())->getTree(
 				NavigationModel::TYPE_MESSAGE,
 				'global-navigation-menu-hubs',
 				[3] // max 3 links
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)